README.md

```diff
@@ -1,79 +1,55 @@
 # kind-local
 
-This is a Proof of Concept of how to run a local container registry in [Kind](https://github.com/kubernetes-sigs/kind) (Kubernetes in Docker).
+This repo documents the best way to run a local [Kind](https://github.com/kubernetes-sigs/kind) cluster with Tilt.
 
 ## Why
```

Comment:
unclear if this section is "why kind" (in which case it's missing info) or "why use a local registry" (in which case it's not titled/prepared super clearly).

Imo it should be "Why KinD + local registry"

Reply:
done